Example/Headers/PhonesHeaderModel.swift

```diff
@@ -17,7 +17,7 @@ struct PhonesHeaderModel {
 extension PhonesHeaderModel: SupplementaryViewModel, CellConvertible {
     typealias Cell = PhonesHeader
 
-    var height: CGFloat {
+    var height: Double {
         return 50
```

Comment:
Niekde som cital (mozno dokonca ty si ma upozornil) ze moze byt pre kompilator efektivnejsie ked priamo vratis `50.0` pretoze uz hned bude vedet ze sa jedna o `Double` (resp. floating point cislo) a ne `Int`.

Reply:
You've got some good points, Bezo!